pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -63,16 +77,34 @@ var (
 		"egressNetworkPolicyName",
 		"egressNetworkPolicyNamespace",
 	}
+	AntreaInfoElementsIPv6 = []string{
```

Comment:
nit: maybe we could define `antreaInfoElements` and use it in the definition of `AntreaInfoElementsIPv4` and `AntreaInfoElementsIPv6` to avoid repetition

Reply:
Do you mean it is better to define 3 lists: `AntreaInfoElements` (common IE), `AntreaInfoElementsIPv4` (IPv4 specific IE) and `AntreaInfoElementsIPv6` (IPv6 specific IE)? Then append them in code below?

Reply:
yes that's what I mean. And the common IE does not need to be exported. https://play.golang.org/p/qDaN_AJzyOP

But if the order of the elements in the array matters, then feel free to ignore this comment